### ISSUE TITLE
The warriors & Naruto

### ISSUE THREAD

**Issue by IrfanH495** [hrydgard/ppsspp#14211]
### What happens? 
The warriors
![Screenshot_20210223-234523](https://user-images.githubusercontent.com/78854859/108878432-d6da3880-7632-11eb-9e16-cd2e0dafdad8.png)
Naruto inpact glich

https://user-images.githubusercontent.com/78854859/108878473-e0fc3700-7632-11eb-9a8f-90861d48776b.mp4

Final fantasy type 0 glich and fps drop if replace textures On

replace textures On
![Screenshot_20210224-000958](https://user-images.githubusercontent.com/78854859/108880695-1c980080-7635-11eb-875b-27d57f6f7587.png)

replace textures Off
![Screenshot_20210224-001104](https://user-images.githubusercontent.com/78854859/108880737-27eb2c00-7635-11eb-80bf-f1ad3d884486.png)

Tenchu san & kurenai, musou orochi 2 special fc 

https://user-images.githubusercontent.com/78854859/108878739-2456a580-7633-11eb-9167-787d89a2f9eb.mp4

Vulkan
Test on v1.11.2-214-ga64737f7f 
### What hardware, operating system, and PPSSPP version?  On desktop and mobile phone, GPU matters for graphical issues.
Sharp & sony xzs
Snapdragon 820
Adreno 530

**Comment by IrfanH495** [comment]
higurashi daybreak portable mega edition fc
And valkyria chronicle 3 glitch I don't know worse than issue https://github.com/hrydgard/ppsspp/issues/14105

https://user-images.githubusercontent.com/78854859/108883001-81545a80-7637-11eb-9a73-c8f1cf3adb11.mp4

**Comment by IrfanH495** [comment]
hatsune miku project diva 2 & extend fc

**Comment by IrfanH495** [comment]
hatsune miku project diva 2 & extend fc
https://drive.google.com/file/d/10vvXOQDTG-4VLvadi6stNrVaFl8yqaaE/view?usp=drivesdk

**Comment by IrfanH495** [comment]
I try too Ppsspp bulid v1.11.2-212-g0cd561384 but still fc

**Comment by IrfanH495** [comment]
I tried again in version v1.11.2-191-g7cefafeb3
Same fc to, Miku & the warriors glich.

**Comment by IrfanH495** [comment]
Last fine in ver ppsspp-v1.11.2-75-gb0fdc7a30-
![Screenshot_20210224-024939](https://user-images.githubusercontent.com/78854859/108899607-33e1e880-764b-11eb-9f2a-ccf5d0996848.png)
![Screenshot_20210224-024834](https://user-images.githubusercontent.com/78854859/108899634-3a706000-764b-11eb-823a-608643a0ca96.png)
![Screenshot_20210224-024900](https://user-images.githubusercontent.com/78854859/108899653-3fcdaa80-764b-11eb-8548-a84dc20f222d.png)

**Comment by unknownbrackets** [comment]
Can you try a few more versions in between?  It seems like the colors are wrong for you too?

For example, v1.11.2-121-gc190abc57 or v1.11.2-87-ga911d99ac might be good to try.

I was thinking it could've been #14207, but you already tried v1.11.2-212-g0cd561384 which was before that.

-[Unknown]

**Comment by IrfanH495** [comment]
v1.11.2-87-ga911d99ac & v1.11.2-212-g0cd561384 good, it's no bug an thare
For v1.11.2-121-gc190abc57 I do not see it
![Screenshot_20210224-093456](https://user-images.githubusercontent.com/78854859/108939054-af5f8c00-7683-11eb-903b-3bbe45ec28f4.png)

sorry 1 wrong writing
Last i try ppsspp-v1.11.2-75-gb0fdc7a30-
sorry if I ask off the mark
not only that, I also couldn't extract the files in the ppsspp apk, it always failed
![Screenshot_20210224-093419](https://user-images.githubusercontent.com/78854859/108938933-7e7f5700-7683-11eb-808d-a064cd54c528.png)

**Comment by unknownbrackets** [comment]
Oh, if v1.11.2-212-g0cd561384 is good then this should be fixed by 579482320.  It didn't fix the issue it was intended to for everyone anyway.

-[Unknown]

**Comment by IrfanH495** [comment]
> Oh, if v1.11.2-212-g0cd561384 is good then this should be fixed by [5794823](https://github.com/hrydgard/ppsspp/commit/579482320d8d4fe4f759b383f0f5cc4a7bd6e41a). It didn't fix the issue it was intended to for everyone anyway.
> 
> -[Unknown]
this issue comes in version v1.11.2-191-g7cefafeb3
for the previous version I don't know
For v1.11.2-212-g0cd561384
I tried it last night it's still in the comments.
I tried new build version v1.11.2-215-g579482320
Fc to.
https://drive.google.com/file/d/11Uzq3ZCXTdjqEkgNTsDvbejuneoJnR8U/view?usp=drivesdk
sorry if my report is not clear.
don't rush, take your time, to avoid fatigue, good job.  I just need to downgrade the PPSSPP.  I'm sorry my english is very bad
